### PR TITLE
adding validation erorr

### PR DESCRIPTION
**Pull request by boo-shehab**
# Hi reviewer 👋

> We make the validation of the email.

- If the email is not in lowercase, it will show them the error message
- If the user tried to submit unveiled information it will not send

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
## Please hold on your project is under review 🛑

File: js/main.js [hunk 1]
```diff
@@ -137,4 +137,23 @@ function pupUpMenuid(id) {
...
+  if (contactFormEmail.value !== contactFormEmail.value.toLowerCase()) {
+    event.preventDefault();
+    document.querySelector('form .erorr-massege').classList.add('active');
+  } else {
+    document.querySelector('form .erorr-massege').classList.remove('active');
+  }
+});
+contactFormEmail.addEventListener('input', () => {
+  if (contactFormEmail.value !== contactFormEmail.value.toLowerCase()) {
+    document.querySelector('form label .erorr-massege').classList.add('active');
```
Comment (Davidosky007) [inline js/main.js]: * According to the [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=When%20the%20user,is%20not%20sent.) the user clicks submit form button before the email is validated and an error message displayed but currently yours displays the error message  shows up before the submit button, Kindly fix this 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @boo-shehab @ifeanyi305 @Moise-code 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Proper use of GitHub flow pattern 👍   

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 1 above)
Comment (boo-shehab) [reply js/main.js]: @Davidosky007 Is it good now?

Comment (Davidosky007) [reply js/main.js]: * Kindly fix the issue pointed out above, I know it's little but adherence to specification is a skill you will need in the future that is why requirements are always given HINT 💭 you can move the email validation into a function and then only call that function when a user tries to submit 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @boo-shehab 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* No linter errors 👍 
* Good looking readme file 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidosky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js (hunk 1 above)
Comment (boo-shehab) [reply js/main.js]: @Davidosky007 why you didn't say that in your first review I don't have an extra review and not I only have one review

File: index.html [hunk 2]
```diff
@@ -195,14 +195,21 @@ <h2>Contact me</h2>
...
+                        <span hidden>put your full name</span>
+                        <input type="text" required name="full-name" id="full-name" placeholder="full name" max-length="30">
+                    </label>
                     
-                    <label for="email" hidden>put your Email</label>
-                    <input type="email" id="email" required name="Email" placeholder="email" max-length="30">
+                    <label for="email">
+                        <span hidden>put your Email</span>
+                        <input type="email" id="email" required name="Email" placeholder="email" max-length="30">
+                        <span class="erorr-massege" aria-live="polite">the Email should be in lower cass</span>
```
Comment (sinansevgi) [inline index.html:206]: [Optional]
- [ ] If you can fix the typo in the error message, your project will look more professional.
```suggestion
                        <span class="erorr-massege" aria-live="polite">the Email should be in lower case</span>
```

**Review by sinansevgi — COMMENTED** [review]
Hello @boo-shehab  👋,

_This is an additional review as a result of the second opinion request._

Looks like you already implemented the requested feature and it is working well ⬇️ 
Screenshot 2022-10-13 at 14-41-19 Student project 1

As this was the only reason your project was not approved **it will be approved** as soon as possible.
Please ask for another review in your Dashboard and the **next Code Reviewer will mark it as approved** right away :)

Keep rocking 🚀!

File: index.html (hunk 2 above)
Comment (roshan-bajgain) [inline index.html:206]: - [ ] [Optional] Everything is great but a small requirement is missing, please implement it If the validation is not OK, you should show an error message to the user near the `Get me Touch` button informing them of the error and the form is not sent.

**Review by roshan-bajgain — APPROVED** [review]
Hi @boo-shehab and @ifeanyi305, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

## Optional suggestions
_Please check the comment below_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**